show/main.py

```diff
@@ -806,6 +806,34 @@ def status(interfacename, verbose):
     run_command(cmd, display_cmd=verbose)
 
 
+
+
+@cli.group(cls=AliasedGroup, default_if_no_args=False)
+def monitortx():
+    """show configurations of monitor tx"""
+    pass
+
+
+# show monitortx tx_config
+@monitortx.command()
+def tx_config():
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    data = config_db.get_table('TX_ERROR_CFG')
+
+    for param in data.keys():
+        value = data[param]['value']
+        if param.lower().endswith('period'):
+            conf = 'period_pool'
+        else:
+            conf = 'threshold'
+
+        click.echo('%s value is %s' % (conf, value))
+
+
+
+
```

Comment:
Can you add show command to show statistics? Is think this is a requirement.

e.g.:

```
admin@sonic:~$ show monitor_tx state
Port                     State
-----------             -------
Ethernet0               Ok
Ethernet4               Not Ok
```